### PR TITLE
Move policy authoring to Antora documentation

### PR DESCRIPTION
**Pull request by zregvart**
Moves the Markdown file `POLICY_AUTHORING.md` and places it in the `antora-docs` directory in Asciidoc format.

### CODE REVIEW

File: antora-docs/modules/ROOT/pages/authoring.adoc [hunk 1]
```diff
@@ -0,0 +1,37 @@
+= Policy Authoring
```
Comment (lcarva) [inline antora-docs/modules/ROOT/pages/authoring.adoc:1]: Let's update the README.md link to this file (or the published one). Under the section "Getting started for policy authors".

Comment (lcarva) [inline antora-docs/modules/ROOT/pages/authoring.adoc:1]: Were there any changes to the content of this file other than formatting?

Comment (zregvart) [reply antora-docs/modules/ROOT/pages/authoring.adoc:1]: The only change was to change the Markdown headings to Asciidoc style headings

**Review by lcarva — APPROVED** [review]

**Comment by zregvart** [comment]
Hmm, only `index.adoc` [gets built](https://github.com/hacbs-contract/ec-policies/runs/7580124498?check_suite_focus=true#step:7:18), odd.

**Pull request merged**